pdns/dnsdistdist/docs/reference/dq.rst

```diff
@@ -217,6 +223,14 @@ This state can be modified from the various hooks.
 
     :param string reason: An optional string describing the reason why this trap was sent
 
+  .. method:: DNSQuestion:setContent(data)
+
+    .. versionadded:: 1.8.0
+
+    Replace the DNS payload of the query with the supplied data, and adjusts the ID in the DNS header to the existing query.
```

Comment:
Example would be nice, especially since it is not 100% clear if the payload should include the header.